how-to/migrate-from-a-previous-version/README.md

```diff
@@ -3,6 +3,59 @@
 > **_:information_source: OpenFin Workspace:_** [OpenFin Workspace](https://www.openfin.co/workspace/) is a commercial product and this repo is for evaluation purposes. Use of the OpenFin Container and OpenFin Workspace components is only granted pursuant to a license from OpenFin. Please [**contact us**](https://www.openfin.co/workspace/poc/) if you would like to request a developer evaluation key or to discuss a production license.
 > OpenFin Workspace is currently **only supported on Windows**.
 
+## Migrate from a previous version - From v8.0 to v9.0
+
+The main focus of this release is:
+
+### New and improved Dock
+
+OpenFin has introduced the ability for developers to register a Workspace Platform with the Dock component. This enables end-users to access multiple Workspace Platforms in a single, unified Dock component.
+
+In addition, Dock is fully themable by Workspace Platforms, to deliver a branded experience whilst being part of a shared experience.
+
+Developers of Workspace Platform can also fully customize the buttons in Dock. For example, they can add, remove and reorder buttons in Dock.
+
+### Change of Behaviour - Home doesn't automatically appear on registration
```

Comment:
I think we can Let this one slide as customiz(s)e was a sacrifice 😜

Reply:
lol took me a while to think what is different. Changed behaviour to the US spelling.